src/components/raycaster.js

```diff
@@ -38,13 +47,14 @@ module.exports.Component = registerComponent('raycaster', {
     this.lineEndVec3 = new THREE.Vector3();
     this.unitLineEndVec3 = new THREE.Vector3();
     this.intersectedEls = [];
-    this.objects = null;
+    this.objects = [];
     this.prevCheckTime = undefined;
     this.prevIntersectedEls = [];
     this.raycaster = new THREE.Raycaster();
     this.updateOriginDirection();
-    this.refreshObjects = bind(this.refreshObjects, this);
-    this.refreshOnceChildLoaded = bind(this.refreshOnceChildLoaded, this);
+    this.setDirty = bind(this.setDirty, this);
```

Comment:
I re-ran some jsperf stuff, we can just start using vanilla bind. I don't think there's perf difference